readme.md

```diff
@@ -292,6 +292,7 @@
 - [Open Source Society University](https://github.com/ossu/computer-science#readme)
 - [Functional Programming](https://github.com/lucasviola/awesome-functional-programming#readme)
 - [Static Analysis & Code Quality](https://github.com/mre/awesome-static-analysis#readme)
+- [Mutation Testing](https://github.com/theofidry/mutation-testing#readme) - Better code by making bugs.
```

Comment:
The description is not really understandable without context. It should be clearer.